src/components/animations/BlockExpand.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="block-expander">
+  <div class="block-expander" :class="isExpanded&&'open'">
```

Comment:
You can use [class style bindings](https://vuejs.org/v2/guide/class-and-style.html) for that

Reply:
added